WcaOnRails/db/structure.sql

```diff
@@ -1238,5 +1238,5 @@ INSERT INTO `schema_migrations` (version) VALUES
 ('20170818164058'),
 ('20170820023104'),
 ('20170823170616'),
-('20170831170616'),
-('20170830140540');
+('20170830140540'),
+('20170831170616');
```

Comment:
What happened here?

Reply:
Running migrations automatically rearrange them by date I think.

Reply:
Oh, haha, I didn't look at that closely enough, thanks, @viroulep!